example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -31,12 +29,10 @@ struct TestBufferKernel
     template<typename TAcc, typename MdSpan>
     ALPAKA_FN_ACC auto operator()(TAcc const& acc, MdSpan data) const -> void
     {
-        auto const idx = alpaka::getIdx<alpaka::Grid, alpaka::Threads>(acc);
-        auto const gridSize = alpaka::getWorkDiv<alpaka::Grid, alpaka::Threads>(acc);
-
-        for(size_t z = idx[0]; z < data.extent(0); z += gridSize[0])
-            for(size_t y = idx[1]; y < data.extent(1); y += gridSize[1])
-                for(size_t x = idx[2]; x < data.extent(2); x += gridSize[2])
+        // Use three nested loops along the dimensions z, y and x
+        for(size_t z: alpaka::uniformElementsAlongZ(acc, data.extent(0))
```

Comment:
I think this should be `uniformElementsZ`, `uniformElementsY` and `uniformElementsX`

Reply:
@fwyzard For accessing via index the corresponding method is called `uniformElementsAlong<>` but for the named version is `uniformElementsZ<>`.
Since you as author of this abstraction did the mistake, is this a sign that we should rename `uniformElementsZ` to `uniformElementsAlongZ` after this PR is merged. 

Currently we have the possibility, after the release renaming is not as easy. Whats your opinion?

Reply:
Which one of names do you prefer ?

Reply:
I have not used the abstraction at the moment but from looking over it I would say `uniformElementsAlongZ` is the better name because `Z` is only replacing the index used if you call `uniformElementsAlong`

Reply:
OK for me, I'll make the change